libraries/botbuilder-dialogs-adaptive-testing/src/testRunner.ts

```diff
@@ -16,34 +16,33 @@ import { TestScript } from './testScript';
  * In charge of running Dialog Adaptive tests.
  */
 export class TestRunner {
-    private resourceExplorer: ResourceExplorer;
-    private testAdapter: TestAdapter;
-
     /**
      * Initializes a new instance of the `TestRunner` class.
      * @param resourcePath Path where the `.dialog` files are located.
      */
-    public constructor(resourcePath: string) {
+    public constructor(private resourcePath: string) {
```

Comment:
@joshgummersall @stevengum Tangential to this PR, is this the conventions we're moving with (access modifier in the ctor arguments vs. instantiating right underneath the class definition)?

I've seen both and am never sure which to use.

Reply:
I'm a big fan of being as succinct as possible. That said, we are not 100% consistent on prefixing private members with underscores which complicates this a bit. In general, I'm in favor of inlining modifiers when we can. In cases where we are prefixing members with underscores, I think it makes the generated method signature docs a bit ugly.